envs/cytominer_env.yaml

```diff
@@ -0,0 +1,10 @@
+name: ctyominer_env
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - python=3.7.0
```

Comment:
don't you think this is overly restrictive? did you mean >=3.7.0?

Reply:
Yes... 🤦 

changed it to `>=3.7` on commit: d74575671a20747cfe0ac21744b162584cc915de